Client/Ecosia/UI/Onboarding/WelcomeTourTransparent.swift

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import Core
+
+final class WelcomeTourTransparent: UIView, NotificationThemeable {
+
+    private weak var stack: UIStackView!
+
+    lazy var currencyNumberFormatter: NumberFormatter = {
+        let formatter = NumberFormatter()
+        formatter.numberStyle = .currency
+        formatter.currencySymbol = "€"
+        return formatter
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+        updateAccessibilitySettings()
+    }
+
+    required init?(coder: NSCoder) {  nil }
+
+    func setup() {
+        let stack = UIStackView()
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        stack.axis = .vertical
+        stack.distribution = .fill
+        stack.alignment = .leading
+        stack.spacing = 8
```

Comment:
Can we make all the new welcome tour screens follow the (annoying) `struct UX`  representation? 🙏

Reply:
Actually just realised that the old ones don't follow it, I think we need to properly define some common suggested patterns for this and other things so that we can slowly refactor whenever needed. 
Personally, I'm not 100% confident of the benefit of having the separate struct, have just done it for the sake of keeping the pattern so far. 
My suggestion would be keeping it as is, maintaining the approach of the other onboarding screens and table a proper code styling discussion for after our GM changes release (already have some other topics in mind as well)